community/values/src/main/java/org/neo4j/values/storable/PointValue.java

```diff
@@ -38,6 +38,8 @@
 
 public class PointValue extends ScalarValue implements Point, Comparable<PointValue>
 {
+    public static String[] ALLOWED_KEYS = new String[]{"crs", "x", "y", "z", "longitude", "latitude", "z", "srid"};
```

Comment:
'z' is duplicated

Reply:
Oh, it should have been "height". Let me fix that.